app/team.go

```diff
@@ -502,8 +501,8 @@ func (a *App) joinUserToTeam(team *model.Team, user *model.User) (*model.TeamMem
 		tm.SchemeAdmin = true
 	}
 
-	etmr := <-a.Srv.Store.Team().GetMember(team.Id, user.Id)
-	if etmr.Err != nil {
+	result, err := a.Srv.Store.Team().GetMember(team.Id, user.Id)
```

Comment:
probably it can be just `rtm, err := ...` and remove the `rtm := result`

Reply:
Can I remove the comment above `rtm := result` ?

Reply:
Yes, I think so